ts/features/payments/history/store/reducers/index.ts

```diff
@@ -74,7 +74,8 @@ const reducer = (
       });
     case getType(walletPaymentHistoryStoreOutcome):
       return updatePaymentHistory(state, {
-        outcome: action.payload
+        outcome: action.payload,
+        success: action.payload === "0" ? true : undefined
```

Comment:
I think I understand where you're coming from with this, but while I see no problem in it, why not simply do this check in a selector?

Reply:
We need to mutate this object because will be used in the Zendesk support payload.